src/audio-worklet/free-queue/src/rollup.config.js

```diff
@@ -0,0 +1,30 @@
+import typescript from "@rollup/plugin-typescript";
```

Comment:
Why do we need this? Can you explain?

Reply:
That is being used for using typescript with rollup.